tests/llmobs/conftest.py

```diff
@@ -191,16 +198,20 @@ def mock_ragas_dependencies_not_present():
 @pytest.fixture
 def ragas(mock_llmobs_span_writer, mock_llmobs_eval_metric_writer):
     with override_global_config(dict(_dd_api_key="<not-a-real-key>")):
-        import ragas
-
+        try:
+            import ragas
+        except ImportError:
```

Comment:
Can we leave a removal note for when we drop support for 3.7. I don't want us to forget to clean up this code.

Reply:
Is this because we don't need to `try/except ImportError` in Python 3.8+?